CRM/Dedupe/Merger.php

```diff
@@ -873,16 +873,17 @@ public static function merge($dupePairs = [], $cacheParams = [], $mode = 'safe',
           unset($dupePairs[$index]);
           continue;
         }
-        CRM_Utils_Hook::merge('flip', $dupes, $dupes['dstID'], $dupes['srcID']);
```

Comment:
These lines relate to a single pair  - so I moved them into the single pair function